docs/DeveloperGuide.md

```diff
@@ -174,6 +174,16 @@ The current implementation thus encapsulates all the interactions with model, wi
 ### Unpool feature
 This feature allows users to remove a pool from the pool list through the specification of an index.
 
+Design considerations include the `findPool` command being able to be used in conjunction with the `unpool` command. For instance, the user might first use `findPool n/Alice` and then followed by `unpool 1`.
+The `findPool n/Alice` command first filters the list of displayed pools, such that only pools in which there is a passenger named Alice will be displayed. Calling the `unpool` command would then remove the pool specified by the provided indices from the currently displayed list, removing it from the system. The `findPool` command works similarly to the `find` command, except that it currently only supports the use of the name prefix: "/n"
+
+The activity diagram below encapsulates the user workflow of adding passengers, finding passengers and then pooling the passengers:
+
+![Activity Diagram for a user using Unpool](images/UnpoolActivityDiagram.png)
```

Comment:
I was contemplating on adding this and decided not to because it would look like a direct copy of the pool activity diagram and an unnecessary repeat

Reply:
I felt that adding this wasn't unnecessary as although this is a similar situation as described in pool, it focuses on a separate set of commands that work similarly well together.